src/components/progress/js/Progress.js

```diff
@@ -132,8 +132,9 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             fluid.progress.updateWidth(that, pixels);
         }
 
-        if (labelText !== null) {
-            fluid.progress.updateText(that.label, labelText);
+        if (fluid.isValue(labelText)) {
+            var text = fluid.stringTemplate(labelText, {percentComplete: percent});
+            fluid.progress.updateText(that.label, text);
```

Comment:
Each update in this `updateProgress` function, such as `updateWidth`, `updateText`, `updateARIA` should be a namespaced model listener. But it's probably ok for now since bringing component code up to date with new infusion features is worth a separate ticket.